inductiva/fluids/simulators/openfoam.py

```diff
@@ -38,3 +38,21 @@ def run(
             openfoam_solver=openfoam_solver,
             n_cores=n_cores,
         )
+
+    def run_async(
+        self,
+        input_dir: types.Path,
+        openfoam_solver: str = "interFoam",
+        n_cores: int = 1,
+    ) -> str:
+        """Run the simulation asynchronously.
+        
+        Args:
+            sim_config_filename: Name of the simulation configuration file.
+            device: Device in which to run the simulation.
+            other arguments: See the documentation of the base class.
+            """
+
+        return super().run_async(input_dir,
+                                 openfoam_solver=openfoam_solver,
+                                 n_cores=n_cores)
```

Comment:
Doesn't this simulator take an `input_filename`? Its this expected?

Reply:
No, openFOAM does not use a single input file to prepare the simulation. In fact it has to do multiple steps to obtain a simulation, which are done on the server side.